pkg/runner/runner_test.go

```diff
@@ -120,6 +120,7 @@ func TestRunEvent(t *testing.T) {
 		{"testdata", "issue-597", "push", "", platforms, ""},
 		{"testdata", "issue-598", "push", "", platforms, ""},
 		{"testdata", "env-and-path", "push", "", platforms, ""},
+		{"testdata", "output", "push", "", platforms, ""},
```

Comment:
Typo?
Your workflow is in folder `outputs` not `output`.

Reply:
Indeed